config.yaml

```diff
@@ -28,8 +28,8 @@ tmpdir: '/tmp'
 kmer_length: 21
 
 # deepvariant
-#DEEPVARIANT_VERSION: '1.3.0'  # CPU-only, will require modifying threads for call_variants rules
-DEEPVARIANT_VERSION: '1.3.0-gpu'  # GPU
+DEEPVARIANT_VERSION: '1.3.0'  # CPU-only, will require modifying threads for call_variants rules
+# DEEPVARIANT_VERSION: '1.3.0-gpu'  # GPU
 N_SHARDS: 256
```

Comment:
Should we cap the shards at 78?  Not sure if it's important.

Reply:
There's a little overhead from spinning up each singularity job, so it's not optimal, but I think that's fine.  The end-user can tweak the shards if they want.